### DISCUSSION TITLE
打开nian都是直接自动跳转到纪念日/倒数日界面吗

### DISCUSSION THREAD

**Discussion by why241207** [WatsonYao/nian_app#126] [Q&A] [answered]
进入或切回nian都是直接自动跳转到纪念日/倒数日界面，还要再手动返回有头像、添加记本的界面，一种打开app广告强制跳转的感觉。可不可直接进到记本列表主页，还是我设置不对?
![02](https://github.com/user-attachments/assets/5a64c108-c71f-4b17-bf54-80475932eaf2)
![0001](https://github.com/user-attachments/assets/1638825c-5e60-4c75-8fe3-cd965a5a1091)

**Comment by WatsonYao** [comment, marked as answer]
用了什么广告跳过之类的app吧，把nian加白名单里面试试。

**Comment by why241207** [comment]
确实是！给力！！谢谢你
